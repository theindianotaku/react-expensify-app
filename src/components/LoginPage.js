import React from 'react';

const LoginPage = () => (
  <div>
    <button>Login</button>
  </div>
);

export default LoginPage;