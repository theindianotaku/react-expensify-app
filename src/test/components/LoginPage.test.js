import React from 'react';
import { shallow } from 'enzyme';
import LoginPage from '../../components/LoginPage';

test('should render LoginPage correctly', () => {
  const wrapper = shallow(<LoginPage />);
  expect(wrapper).toMatchSnapshot();
});