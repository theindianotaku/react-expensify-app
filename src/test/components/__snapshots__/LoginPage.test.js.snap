// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render LoginPage correctly 1`] = `
<div>
  <button>
    Login
  </button>
</div>
`;
